modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -218,6 +223,23 @@ def map_bgs_tracked_items(claim_id)
             }
           end
         end
+
+        def build_supporting_docs(bgs_claim)
+          return [] if bgs_claim.nil?
+
+          docs = evss_docs_service.get_claim_documents(bgs_claim[:benefit_claim_details_dto][:benefit_claim_id]).body
+          return [] if docs.nil? || docs['documents'].blank?
+
+          docs['documents'].map do |doc|
+            {
+              documentId: doc['document_id'],
+              documentTypeLabel: doc['document_type_label'],
+              originalFileName: doc['original_file_name'],
+              trackedItemId: doc['tracked_item_id'],
+              uploadDate: doc['upload_date']
```

Comment:
Since this is a ruby hash, we should really use ruby syntax (snake case) here for the symbol names.
(I realize that it eventually gets converted into a JSON structure, but we should probably stick to ruby convention when working with a ruby object)


FWIW, I notice we've deviated from ruby convention [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb#L140-#L161) as well.  If you wanted to go fix those too, that'd be awesome, but we can also save that for another ticket.